controllers/front/default.php

```diff
@@ -129,7 +129,7 @@ protected function ajaxProcessAddComment()
 			else
 			{
 				$result = false;
-				$errors[] = $module_instance->l('Please wait before posting another comment').' '.Configuration::get('PRODUCT_COMMENTS_MINIMAL_TIME').' '.$module_instance->l('seconds before posting a new comment');
+				$errors[] = $module_instance->l('Please wait before posting another comment', 'default').' '.Configuration::get('PRODUCT_COMMENTS_MINIMAL_TIME').' '.$module_instance->l('seconds before posting a new comment', 'default');
```

Comment:
Hello Jérôme,

We should like other fields add 'default'
https://github.com/PrestaShop/PrestaShop/commit/e0978a0b846a26c2b9e5881f09c509ce224c7e7f
https://github.com/PrestaShop/productcomments/blob/dev/controllers/front/default.php#L75#L86

Thanks